### ISSUE TITLE
Provide option to close tab and open tab

### ISSUE THREAD

**Issue by NivedhaSenthil** [getgauge/taiko#108]
User can switch between tabs and close full browser now but cannot close just one tab.

**Comment by NivedhaSenthil** [comment]
thoughts on API ...
either there can be new api like `close('http://taiko.gauge.org')`  which will take title or url of tab to close and existing `closeBrowser()` will be there as such 

or same `close()` will close the browser and `close('http://taiko.gauge.org')` will close the tab with url.

**Comment by zabil** [comment]
`open` and `close` is generic.

It's better to have separate API's for handling opening and closing of browsers and opening and closing of tabs.

For e.g.
```
openTab("url")
closeTab() // closes current tab
closeTab("url") // close tab with url
```

**Comment by Apoorva-GA** [comment]
Should be available in commit 1af8a9d.

**Comment by sguptatw** [comment]
The issue in close tab is tracked as a part of https://github.com/getgauge/taiko/issues/157 and its dependants.
